src/plugins/iam/components/twitter-app-fields/index.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Field } from 'redux-form';
+import { Card, CardBody, CardHeader } from '@triniti/admin-ui-plugin/components';
+import TextField from '@triniti/cms/components/text-field';
+import TextAreaField from '@triniti/cms/components/textarea-field';
+
+const TwitterAppFields = ({ isEditMode }) => [
+  <Card key="basic">
```

Comment:
@aloncohenn nit: i generally prefer using fragments over keyed array entries these days, you see keys around since fragments werent fully supported when we started this project

Reply:
fixed